src/Pages/class-manage-user/ClassManage.js

```diff
@@ -100,39 +54,96 @@ const initSessionHistory = [
 	},
 ];
 
+
+const StyledHeroImg = styled(HeroImgJr)`
+	display: flex;
+	justify-content: center;
+	align-items: center;
+`;
+
+const StyledClassManage = styled.div`
+	color: ${pr => pr.theme.charcoal};
+	div.main-gal {
+		display: block;
+		height: 95%;
+		width: 90%;
+		background-color: white;
+		h3 {
+			font-size: 3rem;
+			font-weight: 500;
+			line-height: 3.522rem;
+			/* padding: 2rem; */
+		}
+		div.manage-gallery {
+			display: flex;
+			flex-flow: column nowrap;
+			justify-content: space-evenly;
+			align-items: center;
+			height: 90%;
+			h5 {
+				display: block;
+				text-align: center;
+				width: 33.5rem;
+				height: 2.4rem;
+				font-weight: 80rem;
+				font-size: 2rem;
+				line-height: 2.348rem;
+				letter-spacing: 5;
+			}
+			div.class-container {
+				display: flex;
+				flex-flow: row nowrap;
+				justify-content: center;
+				overflow-x: auto;
+				/* overflow-x: scroll; */
+				/* overflow-y: hidden; */
+				width: 90%;
+				height: 50%;
+			}
+
+		}
+	}
+`;
+
 const ClassManage = props => {
-	// const [user, setUser] = useState();
-	// const [activeSessions, setActiveSessions] = useState(initActiveSessions);
-	// const [sessionHistory, setSessionHistory] = useState(initSessionHistory);
-	const [activeSessions, ] = useState(initActiveSessions);
-	const [sessionHistory, ] = useState(initSessionHistory);
+	const [user, setUser] = useState(user26MANY);
+	const [sessions, setSessions] = useState(user["sessions"]);
+	const [activeSessions, setActiveSessions] = useState(user["sessions"]);
+	const [sessionHistory, setSessionHistory] = useState(user["sessions"]);
```

Comment:
I didn't know you could use a state as a initial state for another state.

Reply:
yes indeed! it's just gotta be initialized before!